srcpkgs/The-Powder-Toy/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'The-Powder-Toy'
+pkgname=The-Powder-Toy
+version=94.1
+revision=1
+build_style=scons
+makedepends="SDL2-devel lua51-devel fftw-devel bzip2-devel zlib-devel"
+short_desc="Falling sand physics sandbox, simulates air pressure, velocity & heat"
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="GPL-3.0-or-later"
+homepage="https://powdertoy.co.uk/"
+distfiles="https://github.com/The-Powder-Toy/${pkgname}/archive/v${version}.tar.gz"
+checksum=9770c0732c97e4e89a1e53330c969e9675595f78e6673c519c3724d6df591e72
+CPPDEFINES=IGNORE_UPDATES
+
+case "$XBPS_TARGET_MACHINE" in aarch64*|arm*|ppc*)
+	export make_build_args="--no-sse";;
```

Comment:
No need for export, also pls make this a whitelist with only `x86_64` and `i686`, currently this wouldn't disable SSE for `riscv` or `mips`